python-package/lightgbm/basic.py

```diff
@@ -1748,7 +1756,10 @@ def _load_model_from_string(self, model_str, verbose=True):
             print('Finished loading model, total used %d iterations' % (int(out_num_iterations.value)))
         self.__num_class = out_num_class.value
 
-    def _save_model_to_string(self, num_iteration=-1):
+    def dump_model_to_string(self, start_iteration=0, num_iteration=-1):
+        return self._save_model_to_string(start_iteration, num_iteration)
```

Comment:
Maybe it's better just to rename old private function instead of creating new proxy one?